Criação de plugin db-relational (#18)
stack-spot/graphene-java-api-stack: db-relational-plugin/docs/use-case.md

## **Use Case**

> This Use Case is ideal for starting Java projects with Spring Boot using Maven or Gradle.
> 
> It is possible to apply this plugin to an application that was built from the graphene-java-api-stack.
> 
> In the project folder, just use this command to apply the plugin:

Reviewer: ```suggestion
This use case starts Java projects with Spring Boot using Maven or Gradle.

You can apply this plugin to applications built from the graphene-java-api-stack. See below: 

In the project folder, execute the command:
```

Author: done
